paseos/utils/load_default_cfg.py

```diff
@@ -9,8 +9,10 @@ def load_default_cfg():
     logger.debug("Loading default cfg...")
 
     path = os.path.join(
-        os.path.dirname(__file__) + "/../../resources/", "default_cfg.toml"
+        os.path.dirname(__file__) + "/../resources/", "default_cfg.toml"
     )
+    logger.debug(f"loading path: {path}")
```

Comment:
```suggestion
    logger.debug(f"Loading default cfg from {path}")
```
(can then remove above debugger output in Line9)